### DISCUSSION TITLE
Why is there no arm64? In rancher it is not profitable, i.e. you don't earn any money with it?

### DISCUSSION THREAD

**Discussion by Aramorth** [rancher/rke2#6192] [Q&A]

**Comment by Aramorth** [comment]
open discussion please explain

**Comment by brandond** [comment]
I'm not sure what you're talking about. We do support arm64. What specifically is your question?

**Reply by Aramorth** [reply]
my question is can rke2 be run on raspberry pi 4 arm64 I haven't found anything clear and in a rodeo it was said that you work with k3s but is k3s with web gui or only limited to terminal I would like to use the rancher gui under raspberry pi

**Reply by brandond** [reply]
That doesn't mean that RKE2 won't run on it. RKE2 is much more resource intensive than K3s, and is designed to be run on server-class hardware. If you have low-end hardware like a Pi, K3s is a much better fit.

As to whether we're making money off people running RKE2 on Raspberry Pis... I would guess probably not? But I'm an engineer, not a sales person.

**Reply by brandond** [reply]
> k3s with web gui or only limited to terminal I would like to use the rancher gui under raspberry pi

You can run Rancher on k3s, or rke2, or any other Kubernetes distribution. If you're asking about Rancher, then you're in the wrong place.
